doc/api/fs.markdown

```diff
@@ -522,6 +522,30 @@ to the completion callback. `mode` defaults to `0o777`.
 
 Synchronous mkdir(2). Returns `undefined`.
 
+## fs.mkdtemp(prefix, callback)
+
+Creates a unique temporary directory.
+
+Generates six random characters to be appended behind a required
+`prefix` to create a unique temporary directory.
+
+The created folder path is passed as a string to the callback's second
+parameter.
+
+Example:
+
+```js
+fs.mkdtemp('/tmp/foo-', (err, folder) => {
+  console.log(folder);
+    // Prints: /tmp/foo-itXde2
```

Comment:
See here as for why this indentation was chosen: https://github.com/nodejs/node/pull/5333#discussion_r53561548